src/Microsoft.DocAsCode.Build.Engine/Microsoft.DocAsCode.Build.Engine.csproj

```diff
@@ -28,6 +28,7 @@
     <Compile Include="InvalidPreprocessorException.cs" />
     <Compile Include="CrossReferenceNotResolvedException.cs" />
     <Compile Include="DocumentBuildContext.cs" />
+    <Compile Include="MarkdownProviders\DfmJsonTokenTreeServiceProvider.cs" />
     <Compile Include="MarkdownProviders\JsonTokenTreeServiceProvider.cs" />
```

Comment:
No！  as you can , in the DfmProcess,we use `DfmJsonTokenTreeServiceProvider`

---
In reply to: [77453106](https://github.com/dotnet/docfx/pull/667#discussion_r77453106) [](ancestors = 77453106)